integration-tests/proptest-regressions/dust.txt

```diff
@@ -0,0 +1,7 @@
+# Seeds for failure cases proptest has generated in the past. It is
+# automatically read and these particular cases re-run before any
+# novel cases are generated.
+#
+# It is recommended to check this file in to source control so that
+# everyone who runs the test benefits from these saved cases.
+cc a42eda86ce372dd6468f5624c7cfb708c3a366d82b14fbbb0d724bc18e93c7af # shrinks to 1
```

Comment:
what is this about ?

Reply:
good catch, the test was wrong, fixed, and removed the file!